ui/apps/dashboard/src/components/AppInfoCard/AppInfoCard.tsx

```diff
@@ -29,12 +32,19 @@ type Sync = {
 } & React.ComponentProps<typeof PlatformSection>['sync'];
 
 export function AppInfoCard({ app, className, sync }: Props) {
+  const env = useEnvironment();
   let lastSyncValue;
   if (sync) {
     lastSyncValue = (
       <div className="flex gap-2">
-        <Time value={sync.createdAt} />
-        <SyncStatus status={sync.status} />
+        <SyncStatus status={sync?.status} />
```

Comment:
```suggestion
        <SyncStatus status={sync.status} />
```

Unnecessary optional chaining since this is inside an `if (sync)` block